content/resources/gen3-features.md

```diff
@@ -0,0 +1,81 @@
+---
+identifier: gen3-features
+title: Gen3 features
+linktitle: /resources/gen3-features
+date: 2019-12-04T10:57:21-05:00
+---
+
+
+# A list of Gen3 features
+
+Gen3 is an open-source platform that enables the standing-up of data commons for managing, analyzing, and sharing research data. Below you can find an exhaustive list of Gen3 features.
+
+## Dictionary and metadata
+
+* Explore the graph in the dictionary page of Windmill:
+    * Search keywords
+    * Visualize possible paths between nodes
+    * Download JSON or TSV format for node templates 
+* Explorer project graph and nodes in Windmill:
+    * Visulize graph model
+    * Browse/download data for each node
+* Metadata submission (create, update, delete records) in JSON, TSV or CSV:
```

Comment:
Generally, I think we call "metadata" -> "structured data".  
I don't feel strongly about this, but I was told this a while back by Bob..

Reply:
Good point, replaced metadata with structured data